Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/ReflectedPropertyEditor.cpp

```diff
@@ -733,36 +753,44 @@ namespace AzToolsFramework
                         }
                     }
                 }
+                if ((!node->GetElementEditMetadata()) || (node->GetElementEditMetadata()->m_elementId != AZ::Edit::ClassElements::Group))
```

Comment:
nit: `(!node->GetElementEditMetadata())` can just be `!node->GetElementEditMetadata()` (additional parens)